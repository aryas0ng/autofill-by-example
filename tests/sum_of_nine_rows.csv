11,90
15,85
2,79
5,80
3,79
2,
4,
43,
5,
6,
9,
3,
4,
2,
5,
2,
6,
1,
8,
6,
1,
21,
3,