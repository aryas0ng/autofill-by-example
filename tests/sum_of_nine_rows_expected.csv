11,90
15,85
2,79
5,80
3,79
2,78
4,81
43,79
5,42
6,38
9,40
3,37
4,35
2,52
5,53
2,48
6,46
1,40
8,39
6,31
1,25
21,24
3,3